LRRServer/README.md

```diff
@@ -0,0 +1,4 @@
+# Getting Started
```

Comment:
Please add a short description of the folder, and what it contains.

Reply:
Seconded.